code/src/WijmoProvider/Features/ColumnFilter.ts

```diff
@@ -80,7 +80,14 @@ namespace WijmoProvider.Feature {
         }
 
         public activate(columnID: string): void {
```

Comment:
@joselrio This method should be documented

Reply:
Done!